src/scripts/internal/index.ts

```diff
@@ -3,7 +3,7 @@ export { getRepoInfo } from './api';
 export { formatBytes } from './format';
 export { hashClass } from './crypto';
 export {
-  getSettingsListItem as getNavButtons,
```

Comment:
don't know what heppened here, now we are actually getting the ul of nav buttons